main_spec.tex

```diff
@@ -118,8 +118,8 @@ \subsubsection{\textbf{SHPDEALLC}}\label{subsec:shpdeallc}
 
 \subsection{Communication Management Routines}
 \label{sec:ctx}
-All \openshmem RMA, AMO, and memory ordering routines are
-performed on a communication context.  The communication context defines an
+All \openshmem \ac{RMA}, \ac{AMO}, and memory ordering routines are
+performed on a \newtext{valid} communication context.  The communication context defines an
```

Comment:
Reads a bit funny with "valid", which is adding a restriction.  What about changing "are performed" to "must be performed"?